packages/create-plugin/templates/app/README.md

```diff
@@ -4,9 +4,9 @@ This template is a starting point for building an app plugin for Grafana.
 
 ## What are Grafana app plugins?
 
-App plugins can let you create a custom out-of-the-box monitoring experience by custom pages, nested datasources and panel plugins.
+App plugins can let you create a custom out-of-the-box monitoring experience by custom pages, nested data sources and panel plugins.
 
-## Getting started
+## Get started
```

Comment:
curious about this one. I have mostly always seen (in tech) the term "getting started" in this context, rather than "Get started" and that's why I have always used it in my docs. 

Is there a particular reason why we want to change it here?

Reply:
The short answer is that it's Grafana doc team's official style convention. Simple present tense (like "get") is preferred over gerunds (like "getting") for readability and translatability. Translators, and translating software, have difficulty translating gerunds with accuracy.